### ISSUE TITLE
PHP compile error

### ISSUE THREAD

**Issue by paskuale75** [p2made/yii2-startbootstrap-themes#3]
I follow instructions, but it return this: "Access level to p2m\SB\assets\AgencyAsset::$resourceData must be protected (as in class p2m\assets\base\P2AssetBundle) or weaker"

**Comment by paskuale75** [comment]
Any help for this ? Thanks

**Comment by paskuale75** [comment]
!solved, mistake in repo version, I've removed `"prefer-stable": true,` from composer.json

**Comment by zeejay09** [comment]
Hi, I have the same problem. I quite don't get what you mean by 

>  I've removed "prefer-stable": true, from composer.json

I looked up in the composer.json but nothing there says "prefer-stable" true"

**Comment by zeejay09** [comment]
TLDR; In the CreativeAsset.php or whateever theme you are using, i changed the $resourceData variable's access modifier from private to protected.

Solved. It is because due the variable $resourceData in CreativeAsset.php or whatever theme you are using being private. CreativeAsset.php extends from \p2m\assets\base\P2AssetBundle and P2AssetBundle extends from \p2m\base\assets\P2AssetBase which where the variable $resourceData is originally defined as protected.

So I guess the take-away from all this is that it is not a good idea to try to change visibility of base class instance/members vars in derived classes.
